src/store/reducers/ormReducer/index.js

```diff
@@ -274,6 +296,18 @@ export default function ormReducer (state = {}, action) {
       comment.delete()
       break
 
+    case DELETE_GROUP_RELATIONSHIP: {
+      if (payload.data.deleteGroupRelationship.success) {
+        const gr = GroupRelationship.safeGet({ parentGroup: meta.parentId, childGroup: meta.childId })
+        if (gr) {
+          gr.delete()
+          clearCacheFor(Group, meta.parentId)
```

Comment:
Did these `clearCacheFor`s still prove to be necessary after you solved the ordering issue otherwise? I expect so, but just checking. I was really hoping that the recent ReduxORM update would have made this sorta hack unnecessary.

Reply:
ill double check